### PR TITLE
Added utilities

### PR DESCRIPTION
**Pull request by nishasen**
utilities are storing information or used for retrieving data through calculations

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
Looks Good, Like how you have made small  functions

**Review by kjais1720 — APPROVED** [review]
Everything looks good, great use of utility functions 👍

File: social/src/Utilities/SetDate.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const SetDate = (date) => {
+    const formattedDate = date.split(",")[0].split("/").map(char => char.length===1 ? "0"+char : char);
+    const shiftDate = formattedDate.shift();
+    formattedDate.splice(1, 0, shiftDate);
```
Comment (srejitk) [inline social/src/Utilities/SetDate.js:4]: Good work here!

**Review by srejitk — APPROVED** [review]
Small PR. Not many changes suggested. Good work , clean code.

**Pull request merged**